util/translate_fight.py

```diff
@@ -98,10 +98,18 @@ def build_mapping(translations, ignore_list=[]):
                     # raise Exception('Conflict on %s: "%s" and "%s"' % (default_name, existing_name, name))
                     pass
             else:
-                replace[lang][default_name] = name
+                if default_name and name :
+                    replace[lang][default_name] = name
     return replace
 
 
+def format_output_str(output_str):
```

Comment:
I hadn't added this before, because running `npm run lintfix` will auto-format this properly.

If you're going to do this, can you also add trailing commas on `^\s*]$` and `^\s*}$` too?

Reply:
fixed